docs/hub/spaces-embedding.md

```diff
@@ -0,0 +1,51 @@
+# Embedding your Space
+
+Once you Space is up and running you might wish to embed it in a website or in your blog. 
+Embedding or sharing your Space is a great way to allow your audience to interact with your work and demonstrations without requiring any setup on their side.
+
+## Direct URL
+
+A Space is assigned a unique URL that once the Space is `Running` you can use to share you Space or embed it in a website. 
+This URL is of the form: `"https://<space-subdomain>.hf.space"`. For instance the space [NimaBoscarino/hotdog-gradio](https://huggingface.co/spaces/NimaBoscarino/hotdog-gradio) has corresponding URL of `"https://nimaboscarino-hotdog-gradio.hf.space"`.
+The subdomain is unique and only changes if you move your Space or rename it.
+
+You can find the Space URL along with examples snippets of how to embed your Space directly from the options on your Space page:
+
+<div class="flex justify-center">
+<img class="block dark:hidden" src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/spaces-embed-option.png"/>
```

Comment:
These need to be added

Reply:
you can also use the screenshots from https://gradio.app/sharing_your_app/#embedding-hosted-spaces, IMO

Reply:
Created the PR on the dataset: https://huggingface.co/datasets/huggingface/documentation-images/discussions/13
@julien-c the screenshots available on the gradio doc do not have the black version which is why I use new images.